README.md

```diff
@@ -4,49 +4,56 @@
 | [![Build status](https://ci.appveyor.com/api/projects/status/809xi9ukwo7sgsip?svg=true)](https://ci.appveyor.com/project/adam4813/tec-hem9u) | [![Build Status](https://semaphoreci.com/api/v1/trillek-team/tec/branches/master/shields_badge.svg)](https://semaphoreci.com/trillek-team/tec) |
 
```

Comment:
It might be neat to add a disocrd button to the above table. I am not certain how people have been handling this in other projects.